news/service/Dockerfile

```diff
@@ -8,7 +8,7 @@ ENV FLASK_APP=server.py
 ENV FLASK_DEBUG=1
 ENV FLASK_ENV=development
 EXPOSE 8080
-CMD ["python", "server.py"]
+CMD [ "python", "-m", "flask", "run", "--host", "0.0.0.0", "--port", "8080" ]
```

Comment:
How will this affect dev/hot-reload?

Reply:
still works fine :)

Reply:
Interesting, is `server.py` still used?

Reply:
yeah through env variable `FLASK_APP=server.py`